vaadin-combo-box-flow-demo/src/main/java/com/vaadin/flow/component/combobox/demo/ComboBoxView.java

```diff
@@ -335,6 +413,38 @@ private void customFiltering() {
 
     }
 
+    private void filteringAndSortingWithDataView() {
+        // begin-source-example
+        // source-example-heading: Filtering and Sorting with Data View
+        ComboBox<Person> comboBox = new ComboBox<>("Persons");
+        PersonService personService = new PersonService();
```

Comment:
Maybe just store the collection instead of the service to make it obvious that this is in-memory example

Reply:
Well, I'd better prefer explicitly create a variable and bind it:

```
Collection<Person> persons = personService.fetchAll();
```

because this example requires many `Person` item to represent the filtering and sorting, so I don't really want to make it cumbersome in the example code.